### PR TITLE
added all database/week2 files

### PR DESCRIPTION
**Pull request by rajanibusani**
DB/week2 HW

### CODE REVIEW

File: databases/week2/wk2_part3.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
+--Getting all the tasks assigned to users whose email ends in @spotify.com
+SELECT name AS user, title AS task
+FROM user_task
```
Comment (aina21) [inline databases/week2/wk2_part3.sql:3]: I believe we don't have user_task table, you can use user

File: databases/week2/wk2_part3.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+--Getting all the tasks for 'Donald Duck' with status 'Not started'
+SELECT user.name AS user, title AS task,status.name AS status
+FROM user_task
+INNER JOIN task
+ON user_task.task_id= task.id
+INNER JOIN status
+ON task.status_id = status.id
+INNER JOIN user
+ON user_task.user_id= user.id
+WHERE user.name = 'Donald Duck' AND status.name = 'Not started';
```
Comment (aina21) [inline databases/week2/wk2_part3.sql:20]: I think you only need to join task table to user and status, you don't need  user_task table, 
let's try again, the result should be zero :)

File: databases/week2/wk2_part3.sql [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+--2 rows returned.
+
+--Getting all the tasks for 'Maryrose Meadows' that were created in september.
+SELECT name AS user, title AS task
+FROM user_task
+INNER JOIN task
+ON user_task.task_id= task.id
+INNER JOIN user
+ON user_task.user_id= user.id
+WHERE user.name = 'Maryrose Meadows' AND month(created)=09;
```
Comment (aina21) [inline databases/week2/wk2_part3.sql:30]: I don't have a user_task table, so I can't test your query. you can write with only one join.

**Review by aina21 — CHANGES_REQUESTED** [review]
good job :)
for part4, what is the difference between shipment and order? Do you need product_id in the shipment table?

**Review by aina21 — APPROVED** [review]

**Review by sofiiadidovych — COMMENTED** [review]
Good job! everything is clean and clear! Keep up this way))

**Comment by rajanibusani** [comment]
> Good job! everything is clean and clear! Keep up this way))

@sofiiadidovych Thank you sofia....

**Pull request merged**